crates/re_viewer/src/ui/visible_history.rs

```diff
@@ -0,0 +1,600 @@
+use egui::NumExt as _;
+use re_data_store::{ExtraQueryHistory, VisibleHistory, VisibleHistoryBoundary};
+use re_log_types::{EntityPath, TimeType};
+use re_space_view_spatial::{SpatialSpaceView2D, SpatialSpaceView3D};
+use re_space_view_time_series::TimeSeriesSpaceView;
+use re_types_core::ComponentName;
+use re_viewer_context::{Item, SpaceViewClassName, ViewerContext};
+use re_viewport::Viewport;
+use std::collections::HashSet;
+use std::ops::RangeInclusive;
```

Comment:
We generally try to maintain some order in the import chaos:
```rust
// std

// external

// re_

// crate

// super
```

Reply:
```suggestion
use std::collections::HashSet;
use std::ops::RangeInclusive;

use egui::NumExt as _;

use re_data_store::{ExtraQueryHistory, VisibleHistory, VisibleHistoryBoundary};
use re_log_types::{EntityPath, TimeType};
use re_space_view_spatial::{SpatialSpaceView2D, SpatialSpaceView3D};
use re_space_view_time_series::TimeSeriesSpaceView;
use re_types_core::ComponentName;
use re_viewer_context::{Item, SpaceViewClassName, ViewerContext};
use re_viewport::Viewport;

Reply:
TIL. I thought we just let rustfmt do whatever 😅

Reply:
`rustfmt` will format your imports one block at a time, you get to decide what the blocks look like!